local_build.sh

```diff
@@ -26,9 +26,9 @@ if [[ "$#" -eq "1" ]]; then
         MOUNT="/stackstate-agent-mount"
 
         docker run --rm \
-            -e ARTIFACTORY_USER=$ARTIFACTORY_USER \
-            -e ARTIFACTORY_PASSWORD=$ARTIFACTORY_PASSWORD \
-            -e ARTIFACTORY_URL="artifactory.stackstate.io/artifactory/api/pypi/pypi-local" \
+            -e artifactory_user=$artifactory_user \
+            -e artifactory_password=$artifactory_password \
+            -e ARTIFACTORY_PYPI_URL="artifactory.tooling.stackstate.io/artifactory/api/pypi/pypi-local" \
```

Comment:
This should be `artifactory.tooling.stackstate.io/artifactory/api/pypi/pypi-virtual`.